test/specs/utils/query.js

```diff
@@ -41,6 +41,26 @@ describe('Query class', () => {
 				then.theQueryInstanceShouldHaveAHandlerFor,
 			);
 			describe('#sendRequest', () => {
+				When(
+					'the query instance sends a request and the lisk API instance returns the success response',
+					when.theQueryInstanceSendsARequestAndTheLiskAPIInstanceReturnsTheSuccessResponse,
+					() => {
+						Then(
+							'it should resolve to the result of sending the request',
+							then.itShouldResolveToTheResultOfSendingTheRequest,
+						);
+					},
+				);
+				When(
+					'the query instance sends a request and the lisk API instance returns the fail response',
```

Comment:
`resolves with a failed response` (and corresponding change to the function name).